spec/index.bs

```diff
@@ -3885,17 +3891,23 @@ The following validation rules apply:
   - |destination|.{{GPUTextureCopyView/texture}}.{{GPUTexture/[[textureUsage]]}} must contain
     {{GPUTextureUsage/COPY_DST}}.
 
-  For the texture {{GPUTexture/[[sampleCount]]}}:
-  - |source|.{{GPUTextureCopyView/texture}}.{{GPUTexture/[[sampleCount]]}} must be equal to |destination|.
-    {{GPUTextureCopyView/texture}}.{{GPUTexture/[[sampleCount]]}}.
-  - If |source|.{{GPUTextureCopyView/texture}}.{{GPUTexture/[[sampleCount]]}} is greater than 1:
-    - The copy with |source|, |destination| and |copySize| must be a |copy of the whole subresource|.
-
-  For the texture {{GPUTexture/[[format]]}}:
-  - |source|.{{GPUTextureCopyView/texture}}.{{GPUTexture/[[format]]}} must be equal to |destination|.
-    {{GPUTextureCopyView/texture}}.{{GPUTexture/[[format]]}}.
-  - If |source|.{{GPUTextureCopyView/texture}}.{{GPUTexture/[[format]]}} is a depth-stencil format:
-    - The copy with |source|, |destination| and |copySize| must be a |copy of the whole subresource|.
+  For the |sampleCount|, |source|.{{GPUTextureCopyView/texture}}.{{GPUTexture/[[sampleCount]]}}:
+  - |sampleCount| must be equal to
+    |destination|.{{GPUTextureCopyView/texture}}.{{GPUTexture/[[sampleCount]]}}.
+  - If |sampleCount| is greater than 1:
+    - |copiesWholeSubresources| must be true.
+
+  For the |format|, |source|.{{GPUTextureCopyView/texture}}.{{GPUTexture/[[format]]}}:
+  - |format| must be equal to
+    |destination|.{{GPUTextureCopyView/texture}}.{{GPUTexture/[[format]]}}.
+  - If |format| is a depth-stencil format:
+    - |copiesWholeSubresources| must be true.
+    - |source| and |destination| must refer to the same set of aspects
```

Comment:
This could be somewhere near `copySize`:
```
dictionary GPUTextureCopy {
 required GPUExtent3D size;
 GPUTextureAspect aspect = "all";
};
```

I.e. imagine the opposite - if we had `copySize` as part of the `GPUTextureCopyView`... that would be awful :)

Reply:
I almost wrote up an issue proposing different places to put the aspect (GPUTextureCopyView, GPUDataLayout, or as an extra argument to other methods). I don't think it makes more sense to put it into GPUTextureCopy (which aiui would replace GPUExtent3D in the method signatures?). It makes just as much sense to me next to `mipLevel` and `origin`.

Reply:
sounds reasonable